runtime/oti/j9nonbuilder.h

```diff
@@ -4784,6 +4784,12 @@ typedef struct J9InternalVMFunctions {
 	void ( *setNestmatesError)(struct J9VMThread *vmThread, struct J9Class *nestMember, struct J9Class *nestHost, IDATA errorCode);
 #endif /* J9VM_OPT_VALHALLA_NESTMATES */
 	BOOLEAN ( *areValueTypesEnabled)(struct J9JavaVM *vm);
+	void ( *registerClassLoader)(struct J9JavaVM* javaVM, struct J9ClassLoader* classLoader);
+	void ( *registerClass)(struct J9JavaVM* javaVM, struct J9Class* clazz);
+	void ( *registerCPEntry)(struct J9JavaVM* javaVM, struct J9ClassPathEntry* cpEntry);
```

Comment:
All of these functions should have the "*" next to the variable

Reply:
oh my bad I didn't see the comment before I merged. I'll fix this for next pr